examples/nek5000/make-nek-examples.sh

```diff
@@ -62,7 +62,7 @@ sed -i.bak -e "s|^#FC=.*|FC=\"$FC\"|" \
     -e "s|^#CC=.*|CC=\"$CC\"|" \
     -e "s|^#SOURCE_ROOT=.*|SOURCE_ROOT=\"$NEK5K_DIR\"|" \
     -e "s|^#FFLAGS=.*|FFLAGS+=\"-I../../include\"|" \
-    -e "s|^#USR_LFLAGS+=.*|USR_LFLAGS+=\"-L../../lib -lceed -fopenmp -L/$OCCA_DIR/lib -locca -lm -lcuda\"|" makenek
```

Comment:
@thilinarmtb It should only be necessary to link the (shared) libraries that you call directly. This works for me everywhere that I have tested.

Reply:
I agree, I was being overly cautious.